src/services/account-service.js

```diff
@@ -1,6 +1,40 @@
 import { StorageService } from "./storage";
 
 export class AccountListService {
+  static add(account) {
+    return this.get().then(accounts => {
+      if (!accounts) {
+        this.set([account]);
+      } else {
+        let updatedAccounts = [account].concat(accounts);
+        this.set(updatedAccounts);
+      }
+    });
+  }
+
+  static update(account, amount) {
+    return this.get()
+      .then(accounts => {
+        if (!accounts) {
+          return new Error("empty accounts");
+        } else {
+          return accounts.find(item => {
+            return item.name === account.name;
+          });
+        }
+      })
+      .then(account => {
+        this.get().then(accounts => {
+          let index = accounts.findIndex(item => {
+            return item.name === account.name;
+          });
+          account.balance = parseInt(account.balance) + amount;
+          accounts[index] = account;
```

Comment:
first time you save balance in storage as `string`. After update you save it as `number`, and don't need parse it as integer every time you update balance. so I think it is better to save it in storage as `number` by default:
![image](https://user-images.githubusercontent.com/14272396/37640048-7b94962c-2c1c-11e8-9c3c-38bf04175e92.png)